.github/workflows/pr-checks-dev.yml

```diff
@@ -95,6 +95,9 @@ jobs:
                   password: root
                   host: localhost
 
+            - name: Fix MySQL authentication for PHP < 7.4
+              run: mysql --user="root" --password="root" --database="wordpress_test" --execute="ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY 'root';"
+
```

Comment:
GitHub actions now uses MySQL 8 by default which is not fully compatible with PHP < 7.4.

Reply:
oh wow, oh well, no complaints here